biome.json

```diff
@@ -71,6 +71,16 @@
           }
         }
       }
+    },
+    {
+      "include": ["packages/workbench/src/routes"],
+      "linter": {
+        "rules": {
+          "style": {
+            "useFilenamingConvention": "off"
+          }
+        }
```

Comment:
TanStack Router is very opinionated on file naming, e.g. `$path` is the convention for a dynamic segment. That breaks the Biome rule, so we can just ignore the file naming within the routes directory.